src/main/java/seedu/duke/tasks/Module.java

```diff
@@ -6,21 +6,32 @@
 
 public class Module {
     private static final String LS = System.lineSeparator();
-    private static final String MODULE_STRING_WITH_DESC = "%s (%s)";
+    private static final String MODULE_STRING_WITH_DESC = "%s (%s) (%sMC, Grade: %s)";
+    private static final String MODULE_STRING_NO_DESC = "%s (%sMC, Grade: %s)";
+    private static final String GENERAL_TASK_STRING = "%s";
     private static final String INDENT = StringConstants.INDENT;
+    private static final String NOT_ENTERED = StringConstants.NOT_ENTERED;
+    private static final int NOT_APPLICABLE = 0;
 
-    private String moduleCode;
+    private final String moduleCode;
     private String moduleDescription;
-    private TaskList taskList;
+    private String moduleGrade;
```

Comment:
Wondering if it would be a good idea to make this an enum instead.

Reply:
Yup, I think changing this to an enum is better especially when the grade is used to calculate CAP afterwards, I have added the Grades enum along with its corresponding grade points.